WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.1.inc

```diff
@@ -673,4 +673,7 @@ class WP_Atom_Server {
 	}
 }
 
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] tod
+function tod_something() {}
```

Comment:
If you look at the [action test failure](https://github.com/WordPress/WordPress-Coding-Standards/actions/runs/10708736147/job/29691753365?pr=2479), you'll see that line 677 is NOT generating an error.

Why?

See the note on line 502:

> The errors are triggered on LINE 1 for the unit-test, because it's the phpcs:set command that is wrong, not the implementing code.

You may also see that lines 506-520 also handle the short prefixes stuff already - so consider updating one of the two-character prefixes (e.g. the two emojis) to a three-character prefix, and the `aaa` prefix test on line 514 to be `aaaa`. Then remove this line 676-678, and reset the line 1 error count.

Reply:
gotcha!!!! thankssss